tiproxy/tiproxy-api.md

```diff
@@ -126,3 +126,7 @@ The output is as follows:
 ```bash
 curl http://127.0.0.1:3080/metrics/
 ```
+
+## Access control
+
+You can restrict access to the TiProxy API by enabling TLS via [`server-http-tls`](/tiproxy/tiproxy-configuration.md#server-http-tls). TiProxy then uses the common name in the client certificate to verify access. Alternatively, if you do not use TLS, you can control access using firewall rules.
```

Comment:
The common name verification (cert-allowed-cn) is only supported from v1.4.0-beta.1, which corresponds to the master branch.
See https://docs.pingcap.com/tidb/dev/tiproxy-configuration/#security